dataprofiler/profilers/float_column_profile.py

```diff
@@ -294,7 +293,7 @@ def _get_float_precision(
 
         # Lead zeros: ^[+-.0\s]+ End zeros: \.?0+(\s|$)
         # Scientific Notation: (?<=[e])(.*) Any non-digits: \D
-        r = re.compile(r"^[+-.0\s]+|\.?0+(\s|$)|(?<=[e])(.*)|\D")
+        r = r"^[+-.0\s]+|\.?0+(\s|$)|([e].*)|\D"
```

Comment:
Why is the content changing of this of regular expression

Reply:
it had to be changed since how regex in polars is handled differently than regex in pandas. Polars does not allow for lookahead/behind so it had to be removed. Fortunately, (if I interpreted the regex properly) I don't think the lookahead/behind was needed in the first place.

Reply:
Alright, I am not a regex expert myself, so maybe someone else should look at this, but the reasoning makes sense to me